### ISSUE TITLE
AMA Intake | Create rated issue modal

### ISSUE THREAD

**Issue by suzchap** [department-of-veterans-affairs/caseflow#6981]
## Story
For any AMA forms, when a user needs to add issues, they will be walked through a multi-screen selection process to help them make the best choice. The landing page created in #6966 will be their starting point and a modal on top of it will walk the user through rated, non-rated, and an "unknown" option. 

This ticket is for the first modal after clicking the "+Add issue" button that shows available rated issues. The "Past decisions that are older than one year," the "Notes" field, and other actions listed in AC are out of scope for this issue.

requires #6966 

## Acceptance criteria
- When a user clicks "+Add issue" button on the "Add issue page" (#6966), it overlays a modal showing timely decisions and rating issues
- Options are radio buttons, not checkboxes like before
- It only allows you to select one radio button on the page
- "Add this issue" button is disabled until a radio button is selected (but doesn't need to work yet)
- Copy has been updated to match the Mocks / Copy section

## Notes
- The ratings we show are still the same as before.
- Out of scope for this ticket
   - "Past decisions..." section 
   - "Notes" field
   - "Add this issue" button, when active, doesn't need to work yet
   - Dynamic numbering for "Add issue 1" and "Does issue 1 match..."
   - "Cancel adding this issue" link and the top right corner "X" should be there but don't need to work
   - "None of these match, see more options" button should be there but doesn't need to work
   - Size and scrollability of the modal window

## Copy
- Add issue 1
- Does issue 1 match any of these issues from past decisions?
- Tip: sometimes applicants list desired outcome, not what the past decision was – so select the best matching past decision.
- Past decisions from [MM/DD/YYYY]
- Cancel adding this issue
- None of these match, see more options
- Add this issue

## Mocks
<img width="715" alt="screen shot 2018-09-14 at 11 24 02 am" src="https://user-images.githubusercontent.com/11219417/45559532-b75b4680-b810-11e8-8776-722801c39daf.png">

## How to Validate
- Doesn't require a specific test user
- Form select > search id > review, any answers > on "Add issue page, click "+Add issue"
- TBD: does this need to be tested with all 3 AMA forms or will one suffice?

**Comment by shanear** [comment]
![screen shot 2018-10-01 at 6 28 33 pm](https://user-images.githubusercontent.com/1596259/46319651-d72c9180-c5a8-11e8-8042-81767a262cf8.png)

I can't select an issue 😢

**Comment by leikkisa** [comment]
I think this is possibly handled in the #6982

**Comment by jiahuang** [comment]
Yeah, issue selection wasn't implemented as part of this ticket, only the display. #6982 got in though so you should be able to select if you repull from master

**Comment by leikkisa** [comment]
Going to move this to "Done" since Shane validated this (other than issue selection), and issue selection is coming up in #6982

**Comment by leikkisa** [comment]
@jiahuang @suzchapman
Oh, I'm noticing that the "None of these match" button is missing in the validation.

Suzanne, the Caseflow modal doesn't allow for this, so we'd want to edit that or make a custom modal.

Should we move this back? We could also make a separate (new) ticket, or I could work on it with the non-rating modal (just noticed this so haven't thought about how long it would take, it might be faster for someone else to do in parallel).
